test/cli/bench/run

```diff
@@ -6,8 +6,8 @@ t1_dict=test/cli/bench/t1-dictionary.toml
 t1_in=test/cli/bench/t1-in.psv
 
 t1_expected=test/cli/bench/t1-expected.psv
-t1_out_c=`mktemp -t icicle-bench-t1-c`
-t1_out_psv=`mktemp -t icicle-bench-t1-out`
+t1_out_c=`mktemp -t icicle-bench-t1-c-XXXXXX`
+t1_out_psv=`mktemp -t icicle-bench-t1-out-XXXXXX`
```

Comment:
mktemp is different on os/x / linux, fwiw mafia does this https://github.com/ambiata/mafia/blob/master/script/mafia#L21